conf/evolutions/default/202.sql

```diff
@@ -0,0 +1,20 @@
+# --- !Ups
+ALTER TABLE label_validation ADD source TEXT;
+
+UPDATE label_validation SET source = 'ValidateMobile' WHERE is_mobile = TRUE;
+
+UPDATE label_validation
```

Comment:
Since the Validate page used the "validation" mission type for both desktop _and_ mobile, it looks like this will UPDATE will overwrite the previous one, and the mobile validations will also be marked as 'ValidateDesktop'!

Reply:
Hmm, what if I just reversed the ordering so the 'ValidateDesktop' one occurs first then update the ones to ValidateMobile? Or should I just check if is_mobile is false with the second update query to prevent overwriting the mobile entries?

Reply:
I think that either are fine!